File: src/main/java/ru/dragonestia/apocalypse/item/CopperNuggetItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class CopperNuggetItem extends Item implements AdvItem {

    public CopperNuggetItem(Integer meta, int count) {
        super(ApocalypseID.COPPER_NUGGET, meta, count, "Copper nugget");
    }

    @Override
    public String getRuName() {
        return "Медный самородок";
    }

    @Override
    public String getIcon() {
        return "textures/items/nugget/copper";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/CopperNuggetItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class CopperNuggetItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.COPPER_NUGGET, 0, count, "Copper nugget"), AdvItem {

    override fun getRuName(): String = "Медный самородок"

    override fun getIcon(): String = "textures/items/nugget/copper"

}